src/WebUI/Components/DeploymentsComponent.tsx

```diff
@@ -226,14 +220,17 @@ export class DeploymentsComponent extends BaseComponent<IDeploymentsComponentPro
         return ListComponent.renderColumn(textToRender || "", ListComponent.defaultColumnRenderer, colDataClassName);
     }
 
-    private static _getHeadingContent( deployment:V1Deployment ): JSX.Element {
+    private static _getHeadingContent(deployment: V1Deployment): JSX.Element {
         return (
             <div>
                 <h3>{deployment.metadata.name}</h3>
-                <LabelGroup labelProps={Utils.getUILabelModelArray(deployment.metadata.labels)}
-                    wrappingBehavior={WrappingBehavior.OneLine}
-                    fadeOutOverflow={true}>
-                </LabelGroup>
+                <div className="kube-flex-row">
+                    <span className="secondary-text kind-tag"> Deployment </span>

```

Comment:
why we need a space before and after Deployment, also this is loc issue.

 " Deployment "

 can we deal this with padding/margin?